src/wrappers/wrapper.js

```diff
@@ -120,6 +120,30 @@ export default class Wrapper implements BaseWrapper {
     return true
   }
 
+  /**
+   * Utility to check wrapper is visible. Returns false if a parent element has display: none or visibility: hidden style.
+   */
+  visible (): boolean {
+    if (!this.exists()) {
+      console.log('NOT EXISTS')
```

Comment:
We should throw an error here using the `throwError` function:
`throwError('cannot call visible on a wrapper without an element')`